app/components/traffic-measure/index.js

```diff
@@ -247,6 +282,18 @@ export default class TrafficMeasureIndexComponent extends Component {
 
     //if new save relationships
     if (this.new) {
+      const trafficMeasureResource = yield this.store.findRecord(
+        'resource',
+        TRAFFIC_MEASURE_RESOURCE_UUID
```

Comment:
this seems odd, isn't every traffic measure a resource?

Reply:
.... not sure i kinda just cut and pasted this from the new route

Reply:
@claire-lovisa